support/deploy/run.sh

```diff
@@ -17,7 +17,7 @@ case $2 in
     host="rinkeby.augur.net"
     privateKey=$RINKEBY_PRIVATE_KEY
     gasPrice=20
-    controller="0xa76ecf40e366d3462a857cbb4695158e4601a8f6"
+    #controller="0xa76ecf40e366d3462a857cbb4695158e4601a8f6"
     ;;
```

Comment:
Is this intentionally left commented out?  If so, recommend a comment above it indicating why it is left in but commented out p